### PR TITLE
feat(admin-ui): context details

### PR DESCRIPTION
**Pull request by frdomovic**
# feat(admin-ui): context details
 
## Summary:
Added details card + row items for client keys and users.
TBD -> connect api for fetching client keys and users.

Fixes # -

### CODE REVIEW

File: node-ui/src/constants/en.global.json [hunk 1]
```diff
@@ -126,7 +126,22 @@
...
         "noJoinedAppsListText": "You have not joined any contexts.",
-        "noInviedAppsListText": "You have not been invited to any contexts."
+        "noInviedAppsListText": "You have not been invited to any contexts.",
+        "contextDetails": {
+            "title": "Context Details",
+            "clientKeysListDescription": "Client keys are authorization keys used by the application frontend.",
+            "noClientKeysText": "No client keys found in the context.",
+            "noUsersText": "No users found in the context.",
+            "labelIdText": "Id: ",
+            "labelNameText": "Next: ",
```
Comment (MatejVukosav) [inline node-ui/src/constants/en.global.json]: Next?

File: node-ui/src/constants/en.global.json [hunk 2]
```diff
@@ -126,7 +126,22 @@
...
+            "labelOwnerText": "Owner: ",
+            "labelDescriptionText": "Description: ",
+            "labelRepositoryText": "Repository: ",
```
Comment (MatejVukosav) [inline node-ui/src/constants/en.global.json]: Repository URL

File: node-ui/src/constants/en.global.json (hunk 1 above)
Comment (frdomovic) [reply node-ui/src/constants/en.global.json]: updated to "Name:"

File: node-ui/src/constants/en.global.json (hunk 2 above)
Comment (frdomovic) [reply node-ui/src/constants/en.global.json]: updated

**Pull request merged**